docs-new/docs/graphql/core/actions/action-relationships.mdx

```diff
@@ -135,12 +123,8 @@ Update the definition of the `UpdateAuthorOutput` object type as:
 
 Save the changes and run `hasura metadata apply` to create the
 relationship.
-
-</div>
-
-<div class="tab">
-
-API
+</TabItem>
+<TabItem value="api" label="API">
 
 Action relationships can be added while defining custom types via the
 `metadata_set_custom_types` metadata API:
```

Comment:
Missed Link `metadata_set_custom_types` - [Internal Links section in Guide](https://hge-docs-migration.netlify.app/migration-guide/links#internal-links) for ref.